### PR TITLE
Update Devo Integration

### PR DESCRIPTION
**Pull request by ansfan**
New version of Devo integration with enhanced features. Includes new features for fetch-incidents, querying alerts, multi-table querying, and writing back to Devo. Using newer version of the Devo SDK which has been updated to use Devo's most recent APIs.

## Status
Ready

## Description
New version of Devo integration with enhanced features. Includes new features for fetch-incidents, querying alerts, multi-table querying, and writing back to Devo.

Refer to README.md found here: https://github.com/devods/demisto-devo

## Screenshots
image
image

## Related PRs
List related PRs against other branches:

https://github.com/demisto/dockerfiles/pull/462 <- This is assuming demisto/devo:1.0.0 docker image name to be built.

## Required version of Demisto
4.x.x
5.x.x

## Does it break backward compatibility?
   - Yes
       - We are upgrading this integration to use the newest Devo HTTP APIs. Also using a different authentication method for enhanced security.

## Must have

- [ ] Tests <- **Please advise on documentation on how to write test-playbooks.**
- [X] Documentation (https://github.com/devods/demisto-devo)
- [X] Code Review

## Dependencies
https://github.com/demisto/dockerfiles/pull/462 <- Dockerfiles to create image needed for Devo's Python SDK.

## Additional changes
Re-write of entire existing Devo integration.

## Technical writer review
Mention and link to the files that require a technical writer review.
- [X] [YAML file](https://github.com/demisto/content/compare/master...devods:devo?expand=1#diff-f888861798ba47c4ea6a1552d4848f26)
- [ ] [CHANGELOG](link)

### CODE REVIEW

**Comment by ansfan** [comment]
> ⚠️ We detected security issues in this pull request:
> 
> Hard-Coded Secrets (1)
> Happy with the results? Give your [feedback](https://guardrails.typeform.com/to/BCgBkB).

Guardrails - Base64 encoded image not a secret.

Also for the circleci might have to whitelist the default `devo.com` domains so not to be categorized as secrets.

**Comment by anara123** [comment]
@ansfan Hi. Thank you for the contribution. 
We changed some the way we store our integration. Please take a look at https://github.com/demisto/content/tree/master/docs/package_directory_structure
1. You should run in order to convert yml to package format:
```
PYTHONPATH="`pwd`:${PYTHONPATH}" python3 package_extractor.py -i /Integrations/integration-Devo.yml -o Integrations/Devo -m
```
2. Then run [linting](https://github.com/demisto/content/tree/master/docs/linting)
```
PYTHONPATH="`pwd`:${PYTHONPATH}" python3 Tests/scripts/pkg_dev_test_tasks.py -d Integrations/Devo```

3. Validate files before you commit your changes
```
PYTHONPATH="`pwd`:${PYTHONPATH}" python2 Tests/scripts/validate_files.py
```
4. Add [changelog](https://github.com/demisto/content/tree/master/docs/release_notes) to the package

You can take a look at the following integration example:
https://github.com/demisto/content/tree/master/Integrations/ProofpointTAP_v2

**Comment by ansfan** [comment]
@anara123 Please rebase/fast forward the `demisto:devods_devo_base` branch to the master git commit: `857896ea12c0c360defe1f559390def192fbc4de`. 

I have done this in order to make sure I had the latest testing environment setup. I believe the rebasing is blocking the license/cla check from passing. 

Let me know if this looks okay to you and I also have included pytest unit tests but have set myself to mimic the proofpoint integration by skipping the playbook test.

File: Integrations/Devo/Devo_description.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+*outputs*
+
+| ContextPath       | Type          | Description                                                                                                                                         |
+|-------------------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| Devo.QueryResults | list(objects) | Will return a list of python dictionary objects. Each object in the list will be a row from the query. If no rows are returned result will be null. |
+| Devo.QueryLink    | object        | Will return an JSON object with a single key called DevoTableLink and a URL link back to Devo for the results just queried.                         |
+
+*current caveats*
+- None
```
Comment (anara123) [inline Integrations/Devo/Devo_description.md]: _description file should only contain instructions on how to configure the integration in order to connect to Devo instance.
For each parameter means like you explained above.
Remove the documentation about the commands. For commands you need to create README.md file in this folder and generate documentation file and fill it with data.
Follow: https://github.com/demisto/content/blob/master/docs/integration_documentation/README.MD

**Comment by anara123** [comment]
related issues
https://github.com/demisto/etc/issues/19220
https://github.com/demisto/etc/issues/15672

**Comment by anara123** [comment]
Closing due to inactivity. @ansfan please reopen the PR when you have time to continue the review process

**Pull request closed without merging**